js/Discourse.js

```diff
@@ -329,7 +321,7 @@ class Discourse extends React.Component {
         sound: true,
       });
 
-      addShortcutListener(({userInfo, activityType}) => {
+      addShortcutListener(({userInfo, _activityType}) => {
```

Comment:
why do we keep this param then if we dont use it?

Reply:
It's not needed indeed.